code/obj/item.dm

```diff
@@ -483,6 +487,16 @@
 		src.visible_message("<span class='alert'>[src] catches on fire!</span>")
 		src.burning = 1
 		src.firesource = TRUE
+		if (istype(src, /obj/item/plant))
+			if (!ON_COOLDOWN(global, "hotbox_adminlog", 30 SECONDS)) //blocks every instance of the plant logging
+				for (var/atom/movable/A in get_turf(src))
+					if (istype(A, /obj/item/plant))
```

Comment:
no point in looping through every atom only to then also type check if its a plant, just loop through plants instead.